shared/src/main/scala/Elements.scala

```diff
@@ -436,13 +436,21 @@ object Elements:
       List("filter", "keep-by", "from-base", "10->b"),
       "a: fun, b: lst -> Filter b by truthy results of a",
       "a: lst, b: fun -> Filter a by truthy results of b",
-      "a: num, b: num -> a in base b - list of digits",
-      "a: num, b: str|lst -> a in base with alphabet b",
+      "a: num, b: num -> a from base b to base 10",
+      "a: num, b: str|lst -> a from base with alphabet b to base 10",
     ) {
       case (a: VFun, b) =>
         ListHelpers.filter(ListHelpers.makeIterable(b, Some(true)), a)
       case (a, b: VFun) =>
         ListHelpers.filter(ListHelpers.makeIterable(a, Some(true)), b)
+      case (a: VNum, b) =>
```

Comment:
This also covers `(a: VNum, b: String)`. Is that okay?

Reply:
Exactly as intended.